.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+<!-- Love streamlink? Please consider supporting our collective:
+👉  https://opencollective.com/streamlink/donate -->
```

Comment:
@bastimeyer I agree, I think having it be part of the markdown, as well as in the existing issue template would be better. The existing issue template is pretty important for us.